tx/textual/internal/testpb/1.proto

```diff
@@ -55,3 +55,34 @@ message Bar {
   string bar_id = 1;
   bytes  data   = 2;
 }
+
+// Baz is a sample message type used for testing repeated rendering.
+message Baz {
+  repeated Foo messages   = 1;
+  repeated string string_messages = 2;
+  Vote vote = 3;
+}
+
+enum VoteOption {
+  VOTE_OPTION_UNSPECIFIED = 0;
+  VOTE_OPTION_YES = 1;
+  VOTE_OPTION_ABSTAIN = 2;
+  VOTE_OPTION_NO = 3;
+  VOTE_OPTION_NO_WITH_VETO = 4;
+}
+
+message WeightedVoteOption {
+  // TODO: Enumeration rendering
+  // VoteOption option = 1;
+  string     option = 1;
+  string     weight = 2 [(cosmos_proto.scalar) = "cosmos.Dec"];
+}
+
+message Vote {
+  uint64 proposal_id = 1;
+  // TODO: cosmos.AddressString rendering
+  // string voter       = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string voter       = 2;
+  reserved 3;
+  repeated WeightedVoteOption options = 4;
+}
```

Comment:
I propose to either import these proto definitions from gov, or define dummy structs in this file. Probably the latter is sufficient for the purpose of the tests, but I'm fine to use real-world examples too. Let's just not duplicate these structs

Reply:
I needed to modify them since there wasn't support for enumerations or AddressString

Reply:
ah, I see... done in ac095bd